src/node.Dockerfile

```diff
@@ -1,9 +1,13 @@
 FROM docker.io/cimg/node:current-browsers
 WORKDIR /app
 
+# Install gosu
+USER root
+RUN apt-get update && \
+    apt-get install -y gosu && \
+    rm -rf /var/lib/apt/lists/*
+
 # Install app dependencies
 # A wildcard is used to ensure both package.json AND package-lock.json are copied
 # where available (npm@5+)
-COPY --chown=circleci:circleci package*.json ./
-
-RUN npm install
```

Comment:
Q: Where did npm install go?

Reply:
npm install is also in the command script, run_node_watch.sh. So redundant IMO.